versions.json

```diff
@@ -823,6 +823,24 @@
         ],
         "fpm_directories": ["include","lib"]
     },
+    "jsoncons": {
+        "git_repository": "https://github.com/danielaparker/jsoncons.git",
```

Comment:
NTS: Remember to fork this repo into the iRODS namespace once this PR is ready for merging.

Reply:
forked it.    can easily sync it back up if it gets to 1.0.0 soonish...

Reply:
https://github.com/irods/jsoncons